pkg/apis/migration/migration.go

```diff
@@ -325,10 +325,11 @@ func (volumeMigration *volumeMigration) registerVolume(ctx context.Context, volu
 		log.Errorf(msg)
 		return "", errors.New(msg)
 	}
-	datastoreName := re.FindAllString(volumeSpec.VolumePath, -1)[0]
-	vmdkPath := strings.TrimSpace(strings.Trim(volumeSpec.VolumePath, datastoreName))
-	datastoreName = strings.Trim(strings.Trim(datastoreName, "["), "]")
-
+	datastoreFullPath := re.FindAllString(volumeSpec.VolumePath, -1)[0]
```

Comment:
In the notes, you have mentioned https://play.golang.org/p/n5qimFHvgjV and it is using regular expression, but here in the code, I do not see it. Are you planning to add regex here?

Reply:
Don't we already have a regex in Line: 322 ? I think the `https://play.golang.org/p/n5qimFHvgjV` is also using the same regexp, right

Reply:
Thanks. Did not see in the diff. So I thought we have a different implementations here.